projects/step-core/src/lib/services/plan-dialogs.service.ts

```diff
@@ -118,7 +119,11 @@ export class PlanDialogsService implements PlanLinkDialogService {
   private openPlanInternal(planEditLink: string, artefactId?: string): void {
     const queryParams = artefactId ? { [ARTEFACT_ID]: artefactId } : undefined;
     const commands = planEditLink.split('/');
-    this._router.navigate(commands, { queryParams });
+    if (!this._router.url.includes(EDITOR_URL)) {
+      this._router.navigate(commands, { queryParams });
+      return;
+    }
+    from(this._router.navigateByUrl('/')).subscribe(() => this._router.navigate(commands, { queryParams }));
```

Comment:
@lucianbaciu 
`skipLocationChange` adds another bug. On the back button, the location was changed but UI hasn't been updated.
So in case of plan editor, I solve the issue in another way.

Please note, that I have to bring back two legacy templates, that have been removed during the clean-up, because they are actually using.